argo/argo-cluster-generation-from-rancher-capi/argo-cluster-generation-from-rancher-capi.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: argo-cluster-generation-from-rancher-capi
+  annotations:
+    policies.kyverno.io/title: Argo Cluster Secret Generation From Rancher CAPI Secret
+    policies.kyverno.io/category: Argo
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Cluster
```

Comment:
Subject here looks like it would be "Secret" since that's what you're matching on and also what you're generating.

Reply:
Will do